spk/python3/Makefile

```diff
@@ -1,7 +1,7 @@
 SPK_NAME = python3
-SPK_SHORT_VERS = 3.6
-SPK_VERS = $(SPK_SHORT_VERS).10
-SPK_REV = 11
+SPK_SHORT_VERS = 3.7
+SPK_VERS = $(SPK_SHORT_VERS).7
+SPK_REV = 1
```

Comment:
SPK_NAME is still python3, so we must increment and use SPK_REV = 12

Reply:
Thanks for the tip. I thought the revision was "on top" of the short version not parallel.